conn.go

```diff
@@ -900,7 +900,11 @@ func (c *Conn) queueRequest(opcode int32, req interface{}, res interface{}, recv
 	switch opcode {
 	case opClose:
 		// always attempt to send close ops.
-		c.sendChan <- rq
+		select {
+		case c.sendChan <- rq:
+		case <-time.After(time.Second):
+			c.logger.Printf("gave up trying to send opClose to server")
```

Comment:
We should probably find a way to send an error back to the Close() call

Reply:
We could, by means of writing an error to the (buffered) `rq.recvChan`; however, in the current API version the `Close()` function does not return any error at all. Hence, the simple thing is still nominations 🥇